dashboard/components/LoadingSpinner.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="loading-spinner" :class="{ 'loading-spinner-small': small }" v-if="loading">
+		<div class="loading-spinner-rect1"></div>
+		<div class="loading-spinner-rect2"></div>
+		<div class="loading-spinner-rect3"></div>
+		<div class="loading-spinner-rect4"></div>
+		<div class="loading-spinner-rect5"></div>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { defineProps } from 'vue';
+
+defineProps( {
```

Comment:
with the new script setup syntax it might be a good idea for the future to define the props with a const like this:
```suggestion
const props = defineProps( {
```
This way we can use `props.loading` in the code/template and distinguish better between props and local variables. Might improve readability of the code.

In this particular case the `LoadingSpinner` component is a very small component that doesn't do much so I think this is perfectly fine and you don't have to change it, I just wanted to mention this thought.